ports/stduuid/vcpkg.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "stduuid",
+    "version-string": "1.0-5890c94",

```

```suggestion
    "version-date": "2021-02-23",
```
We may use the date that you accept the commit as version here. Please see https://github.com/microsoft/vcpkg/blob/2020.11/docs/maintainers/manifest-files.md#version-string.

For field name, please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions.